### ISSUE TITLE
Potential false positive in "Unused variables should be removed"

### ISSUE THREAD

**Issue by kenherring-rkt** [Riverside-Software/sonar-openedge#1019]
When using `display ttTable.field @ vOthervar in frame vFrame` we get an issue for unused variable on `vOthervar` if it's not used for anything else.  In this case it's not really unused, but instead acts as a placeholder in the frame.

## Sample code, with workaround

```progress
define variable cName as character no-undo.

define temp-table ttPeople no-undo
  field name as character.

form
  cName label "Name" at 1 format "x(30)"
  with frame fName.

run initTable.
run displayName.
return.

procedure initTable:
  create ttPeople.
  assign ttPeople.name = "My Name".
  release ttPeople.
end procedure.

procedure displayName:
  find first ttPeople no-lock no-error.

  //false positive code smell
  display ttPeople.name @ cName
      with frame fName.
  
  //alternative option w/o code smell
  // assign cName = ttPeople.name.
  // display cName
  //     with frame fName.
end procedure.
```

## Sonar issue screenshot:

![image](https://user-images.githubusercontent.com/87667618/207473400-dba590c0-5401-423c-9f76-01d86c59603a.png)

## Version info

* SonarQube Server: 9.7.1
* sonar-openedge: 2.17.1

**Comment by gquerret** [comment]
Confirmed

**Comment by gquerret** [comment]
Check skipVarInFrames rule property to skip those issues.

**Comment by gquerret** [comment]
See #383
